lib/utils/string_extension.dart

```diff
@@ -10,5 +10,8 @@ extension StringExtension on String{
       return null;
     }
   }
+  
+  String get symbolFromAmount { return split(" ")[1]; }
+
```

Comment:
Amount string coming from the chain is fixed and always looks like this:

"1.2345 SEEDS"

The number of decimals is _always_ fixed to 4 in our case - because the on chain token "SEEDS" is defined as having 4 digits of precision.

So would equally look like this for a value of 1:
"1.0000 SEEDS"

There's always a space after the number, and the token ticker symbol is a string after that. Will work for any token amount string.